README.md

```diff
@@ -62,7 +71,7 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-Your answer...
+It allows us to export code from one file to another.   We use it to separate our js files by exposing their content as one global variable.
 ```
 
```

Comment:
I realize there is portion of express lesson plan talks about these being globals but this technically wrong in the context of Node. Apologies for the confusion.